src/styles/index.scss

```diff
@@ -1,3 +1,9 @@
+@import 'parts/card';
+@import 'parts/stars';
+@import './utils/mixins';
+
 body {
   margin: 0;
+  padding: 20;
```

Comment:
The padding value should have a unit. Consider using 'px' or another appropriate unit for the padding value.